app/controllers/sessions_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
         redirect_to root_url, notice: t('.session_created')
       else
         flash.now[:alert] = t('.activate_account')
-        redirect_to account_activation_url(user.id)
+        redirect_to account_activation_url(user.id)   
```

Comment:
I'm a whitespace nazi and I see some whitespace at the end of this line that doesn't belong there.